### DISCUSSION TITLE
Can I use THM module in my own app?

### DISCUSSION THREAD

**Discussion by Ethan-xj** [idaholab/moose#24083] [Q&A General] [answered]
@joshuahansel @GiudGiud The codes I wrote these days are all in THM document. I was going to create a new app but I found there was no switch to turn on THM module in the Makefile. I want to know that if I can use THM module in my own app now?

Ethan

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can add that switch in the Makefile.
Add: `THERMAL_HYDRAULICS          := yes`
next to the other modules

Guillaume

**Reply by Ethan-xj** [reply]
OK, I think this need me to update the MOOSE version.
